receiver/redisreceiver/converters.go

```diff
@@ -0,0 +1,29 @@
+package redisreceiver
+
+import (
+	"strconv"
+
+	metricsProto "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
```

Comment:
We usually alias this import to `metricspb` elsewhere in the codebase.
```suggestion
	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
```

Reply:
Fixed.